meteor/client/lib/MeteorReactComponent.ts

```diff
@@ -49,8 +49,9 @@ export class MeteorReactComponent<IProps, IState = {}> extends React.Component<I
 		return computation
 	}
 	subscriptionsReady(): boolean {
-		for (let sub of Object.values(this._subscriptions)) {
-			if (!sub.ready()) {
+		const values = Object.values(this._subscriptions)
```

Comment:
I'd prefer variable naming to be more descriptive, eg "subscriptions" is better than "values" imo.

Reply:
Renamed to `subscriptions`.